angular-spa-ui/js/search-app/search-app-ctrl.js

```diff
@@ -27,41 +27,48 @@ module.exports = function(app) {
 
             function getToken(data) {
                 for (var x in data){
-                    if (data[x].url && (data[x].url === path.getTokenPath)){
+                    if (data[x].url && (data[x].url === paths.tokenPath)){
                         userData.userToken = data[x].data.token;
                     }
                 }
             }
 
-            path.getValidatePath = path.getValidatePath + userData.userId + '/' + userData.userToken;
+            paths.validatePath += userData.userId + '/' + userData.userToken;
         };
 
-        promises.getAsyncData(methods.getMethod, path.getUserPath)
+        promises.getAsyncData(methods.GET, paths.userPath)
             .then(
                 function(data) {
                     initUserData(data);
-                    return promises.getAll(methods.getMethod, [path.getRolePath, path.getTokenPath] )
+                    return promises.getAll(methods.GET, [paths.rolePath, paths.tokenPath] )
                 })
             .then(
                 function(data) {
                     initRTData(data);
-                    return promises.getAsyncData(methods.getMethod, path.getValidatePath);
+                    return promises.getAsyncData(methods.GET, paths.validatePath);
                 })
             .then(
                 function(data) {
-                    userData.userValidate = data.data.result;
-                    var element = angular.element( document.querySelector( '#main' ) );
-                    getTemplate.getByTrustedUrl('/main.html', element, $scope);
-                    // all good
+                    userData.userValidateResult = data.data.result;
+                    var element = angular.element(document.getElementsByTagName('header'));
+                    getTemplate.getByTrustedUrl('/header.html', element, $scope)
+                    .then(
+                        function() {
+                        element = angular.element(document.getElementsByTagName('footer'));
+                        return getTemplate.getByTrustedUrl('/footer.html', element, $scope);
+                    })
+                    .catch(
+                        function(err) {
+                        console.log(err);
+                    });
                 })
             .catch(
-                function(error){
+                function(error) {
```

Comment:
Why are you ignoring codestyle?
function_____a(variable)_____{
...
var_____b_____=_____function_____(...)_____{